src/Parlot/Fluent/ParseContext.cs

```diff
@@ -4,7 +4,9 @@ namespace Parlot.Fluent
 {
     public class ParseContext
     {
-        public static int DefaultCompilationThreshold = 0;
+#pragma warning disable CA2211 // Non-constant fields should not be visible
+        public static int DefaultCompilationThreshold;
```

Comment:
Need to check that. Maybe the `0` is something I forgot to change. And also I don't see why this couldn't be `const` (without checking the rest of the code).